src/transforms/tokenizer.rs

```diff
@@ -98,16 +99,16 @@ impl Transform for Tokenizer {
         let value = event.as_log().get(&self.field).map(|s| s.to_string_lossy());
 
         if let Some(value) = &value {
-            for ((name, conversion), value) in self.field_names.iter().zip(parse(value).into_iter())
+            for ((path, conversion), value) in self.field_names.iter().zip(parse(value).into_iter())
             {
                 match conversion.convert(value.as_bytes().into()) {
                     Ok(value) => {
-                        event.as_mut_log().insert(name.clone(), value);
+                        event.as_mut_log().insert_path(path.clone(), value);
                     }
                     Err(error) => {
                         debug!(
                             message = "Could not convert types.",
-                            name = &name[..],
+                            path = ?path,
```

Comment:
This isn't ideal log output, since it'll show the type structure. Probably not a blocker, but something we should make sure to address before too long.